src/coreComponents/mesh/EdgeManager.hpp

```diff
@@ -290,7 +290,7 @@ class EdgeManager : public ObjectManagerBase
    * @brief Return the  maximum number of edges per node.
    * @return Maximum allowable number of edges connected to one node (hardcoded for now)
    */
-  constexpr int maxEdgesPerNode() const { return 100; }
+  constexpr int maxEdgesPerNode() const { return 200; }
```

Comment:
This is a really large number! It seems too large to be required. @francoishamon Can you comment on the possibility that there are actually more than 100 edges per node?

Reply:
I don't see what geological feature or topology could require so many edges/faces per node, even for an unstructured mesh. I would say that this error reveals either a problem in the mesh, or in the mesh importer that we need to investigate. If the mesh actually has a node with more than 100 edges/faces per node, the mesh can probably be optimized to avoid this situation.

Reply:
With one rank I get:
```
Rank 0: maxNodeEdges = 45
Rank 0: maxNodeFaces = 125
```
With two ranks I get:
```
Rank 0: maxNodeEdges = 45
Rank 1: maxNodeEdges = 41

Rank 0: maxNodeFaces = 125
Rank 1: maxNodeFaces = 114
```
With sixteen ranks I get:
```
Rank 0: maxNodeEdges = 42
Rank 1: maxNodeEdges = 45
Rank 2: maxNodeEdges = 41
Rank 3: maxNodeEdges = 43
Rank 4: maxNodeEdges = 36
Rank 5: maxNodeEdges = 38
Rank 6: maxNodeEdges = 40
Rank 7: maxNodeEdges = 42
Rank 8: maxNodeEdges = 40
Rank 9: maxNodeEdges = 38
Rank 10: maxNodeEdges = 40
Rank 11: maxNodeEdges = 41
Rank 12: maxNodeEdges = 38
Rank 13: maxNodeEdges = 41
Rank 14: maxNodeEdges = 37
Rank 15: maxNodeEdges = 36

Rank 0: maxNodeFaces = 116
Rank 1: maxNodeFaces = 125
Rank 2: maxNodeFaces = 114
Rank 3: maxNodeFaces = 120
Rank 4: maxNodeFaces = 98
Rank 5: maxNodeFaces = 105
Rank 6: maxNodeFaces = 111
Rank 7: maxNodeFaces = 115
Rank 8: maxNodeFaces = 109
Rank 9: maxNodeFaces = 108
Rank 10: maxNodeFaces = 111
Rank 11: maxNodeFaces = 113
Rank 12: maxNodeFaces = 104
Rank 13: maxNodeFaces = 114
Rank 14: maxNodeFaces = 101
Rank 15: maxNodeFaces = 98
```

Which are all the same as the single rank case so I'm not sure exactly why it worked there. Either way 125 faces seems like a lot for a single node.

Reply:
Yes, I agree. Thanks for investigating this. I am going to see with @AntoineMazuyer if there is a way to modify the mesh generation step to make sure that we don't end up with a mesh that has so many faces per node.

Reply:
I would classify that as an "insane" number of faces for a node.